packages/dd-trace/test/plugins/util/web.spec.js

```diff
@@ -751,138 +749,7 @@ describe('plugins/util/web', () => {
-    let app
-    let port
-    let server
-    let http
-
-    beforeEach(done => {
-      agent.load('express')
-        .then(getPort)
-        .then(_port => {
-          port = _port
-          http = require('http')
-          express = require('express')
-          app = express()
-
-          server = app.listen(port, '127.0.0.1', () => done())
-        })
-    })
-
-    afterEach(done => {
-      agent.close().then(() => {
-        server.close(() => done())
-      })
-    })
-
-    it('should run res.end handlers in the request scope', done => {
-      let handler
-
-      const interval = setInterval(() => {
-        if (handler) {
-          handler()
-          clearInterval(interval)
-        }
-      })
-
-      app.use((req, res) => {
-        const end = res.end
-
-        res.end = function () {
-          end.apply(this, arguments)
-
-          try {
-            expect(tracer.scope().active()).to.not.be.null
-            done()
-          } catch (e) {
-            done(e)
-          }
-        }
-
-        handler = () => res.status(200).send()
-      })
-
-      const req = http.get(`http://127.0.0.1:${port}`)
-      req.on('error', done)
-    })
-
-    it('should run res.end handlers in the request scope for clones', done => {
-      let handler
-
-      const interval = setInterval(() => {
-        if (handler) {
-          handler()
-          clearInterval(interval)
-        }
-      })
-
-      app.use((req, res) => {
-        const clone = Object.create(res)
-
-        clone.end = function () {
-          res.end.apply(this, arguments)
-
-          try {
-            expect(tracer.scope().active()).to.not.be.null
-            done()
-          } catch (e) {
-            done(e)
-          }
-        }
-
-        handler = () => clone.end()
-      })
-
-      const req = http.get(`http://127.0.0.1:${port}`)
-      req.on('error', done)
-    })
-
-    it('should run "finish" event handlers in the request scope', done => {
-      app.use((req, res, next) => {
-        res.on('finish', () => {
-          try {
-            expect(tracer.scope().active()).to.not.be.null
-            done()
-          } catch (e) {
-            done(e)
-          }
-        })
-
-        res.status(200).send()
-      })
-
-      const req = http.get(`http://127.0.0.1:${port}`)
-      req.on('error', done)
-    })
-
-    it('should run "close" event handlers in the request scope', done => {
-      const sockets = []
-
-      app.use((req, res, next) => {
-        res.on('close', () => {
-          try {
-            expect(tracer.scope().active()).to.not.be.null
-            done()
-          } catch (e) {
-            done(e)
-          }
-        })
-
-        sockets.forEach(socket => socket.destroy())
-      })
-
-      server.on('connection', (socket) => {
-        sockets.push(socket)
-      })
-
-      const req = http.get(`http://127.0.0.1:${port}`)
-      req.on('error', () => {})
+      expect(tags).to.have.property(ERROR, false)
```

Comment:
Removing all the tests here because they're effectively duplicates?

Reply:
Yes, they were basically testing Express more than anything.